lib/pushmi_pullyu/queue.rb

```diff
@@ -0,0 +1,53 @@
+require 'redis'
+
+# Quick and dirty take on sorted sets
+
+# 1) Create a sorted set in Redis (https://redis.io/topics/data-types). Call it preservation_queue
+#
+# 2) In GenericFile add an after_save that:
+#    - determines a monotonically increasing "score". Obvious scores would be either the time in seconds/milliseconds
```

Comment:
Just to clarify the score could just be a timestamp (in epoch time for example)?  Or would something more complicated be required for this?